qa/smoke-test-http/src/test/java/org/opensearch/http/IdentityAuthenticationIT.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.http;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collection;
+import org.opensearch.client.Request;
+import org.opensearch.client.RequestOptions;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.util.FeatureFlags;
+import org.opensearch.identity.shiro.ShiroIdentityPlugin;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.rest.RestStatus;
+import org.opensearch.test.OpenSearchIntegTestCase;
+import org.opensearch.transport.Netty4ModulePlugin;
+import org.opensearch.transport.nio.NioTransportPlugin;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.core.StringContains.containsString;
+
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.SUITE, supportsDedicatedMasters = false, numDataNodes = 1)
+public class IdentityAuthenticationIT extends HttpSmokeTestCase {
```

Comment:
Is there a way to have this test inside the identity-shiro module?

Reply:
Ehhh, it is not looking good... Basically, if you have this as part of the module the node will not close after the tests execute. So you will successful auth but then the nodes stay open and the test fails as part of the closing process. I tried narrowing down the cause using a debugger and stepping through but it is incredibly complicated to follow the steps that the node takes when executing the requests.

Is there a reason we cannot/should not keep this in the normal test suite? I moved it here since you can add whatever settings you want as part of the Setup step for these tests so I was able to get the tests to work correctly this way.

Reply:
The cluster continues to hang when we try to separate out the basic auth tests into the module itself. Is there an alternative to moving the basic auth tests into the module that will work?